content/team/_index.md

```diff
@@ -73,10 +77,17 @@ SkyWalking have hundreds of Contributors, you could find them in our repository
 - [SkyWalking kubernetes](https://github.com/apache/skywalking-kubernetes/graphs/contributors)
 - [SkyWalking K8s Operator - SkyWalking Cloud on Kubernetes](https://github.com/apache/skywalking-swck/graphs/contributors)
 - [SkyWalking Python Agent](https://github.com/apache/skywalking-python/graphs/contributors)
+- [SkyWalking CLI](https://github.com/apache/skywalking-cli/graphs/contributors)
+- [SkyWalking Client JS](https://github.com/apache/skywalking-client-js/graphs/contributors)
+- [SkyWalking satellite](https://github.com/apache/skywalking-satellite/graphs/contributors)
+- [SkyWalking agent test tool](https://github.com/apache/skywalking-agent-test-tool/graphs/contributors)
+- [SkyWalking NodeJS Agent](https://github.com/apache/skywalking-nodejs/graphs/contributors)
+- [SkyWalking Rust Agent](https://github.com/apache/skywalking-rust/graphs/contributors)
```

Comment:
No one is working on Rust, as the community's C++ SDK is going to be ready soon, I think I has no plan back to this :)

Reply:
Ok, i will delete it